src/core/Akka.MultiNodeTestRunner.Shared/Sinks/SinkCoordinator.cs

```diff
@@ -142,13 +142,14 @@ private void InitializeReceives()
                 TotalReceiveClosedConfirmations = Sinks.Count;
                 ReceivedSinkCloseConfirmations = 0;
 
+                var self = Self;
                 foreach (var sink in Sinks)
                 {
                     sink.RequestExitCode(Self);
                     sink.Close(Context.System)
                         .ContinueWith(r => new SinkClosed(),
                         TaskContinuationOptions.ExecuteSynchronously)
-                        .PipeTo(Self);
+                        .PipeTo(self);
```

Comment:
Good to know, but I was just changing it to closures everywhere rather than leave it up to the next contributor's TPL knowledge.